package/upgrade/upgrade_manifests.sh

```diff
@@ -1106,6 +1106,84 @@ pause_all_charts() {
   done
 }
 
+skip_restart_rancher_system_agent() {
+  plan_manifest="$(mktemp --suffix=.yaml)"
+  plan_name="$HARVESTER_UPGRADE_NAME"-skip-restart-rancher-system-agent
+  plan_version="$(openssl rand -hex 4)"
```

Comment:
Just a suggestion, would it be preferred if we used a predictable plan version (e.g. upgrade name + target version)? So even if the upgrade pod is rebuilt for unexpected reasons, it will try to reuse the existing plan, and leaving no legacy plans behind.

Reply:
IIRC, if reusing the plan version and the plan's content (the script itself) is not changed, no re-execution will happen because a hashed value for the plan was already added to the nodes' label if it has been executed before. This is just a one-time plan. We'll remove it after execution, so a random string for the plan version would be good enough.

Reply:
In that case, we don't need to change the existing one, thanks.